frontend/public/components/utils/resource-metrics.tsx

```diff
@@ -62,6 +62,13 @@ export const ResourceMetricsDashboard: React.FC<ResourceMetricsDashboardProps> =
             title={t('public~Network out')}
           />
         </GridItem>
+        <GridItem xl={6} lg={12}>
+          <ResourceMetricsDashboardCard
+            namespace={obj.metadata.namespace}
+            queries={queries[ResourceUtilizationQuery.CPU_ARCH]}
+            title={t('public~CPU Architecture')}
```

Comment:
```suggestion
            title={t('public~CPU architecture')}
```

Reply:
Thank you, we'll work on it

Reply:
Thanks for the suggestions , Modified the code as suggested above.